### PR TITLE
Implement template emails for the minimal theme

### PR DESCRIPTION
**Pull request by ericvalcik**

### CODE REVIEW

**Comment by jvorcak** [comment]
I think we should try to follow DRY and make components out of reusable parts e.g. Footer.

The user story is that user downloads the template and should only do relevant changes in specific components instead of modifying all emails just to update footers.

**Review by iamhectorsosa — COMMENTED** [review]
We should also try to demonstrate the modularity of using our library to create templates that function as React components.

Let's not forget to add props to these examples and perhaps create defaults to showcase this.

`send(<BookingConfirmation name="Eric" bookingId="420" />);`

File: src/emails/booking-confirmation.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+              requests or requirements, please do not hesitate to contact our
+              front desk team.
+            </Text>
+            <Text>
+              Please note that a deposit may be required upon check-in and any
+              additional charges will be applied to your final bill upon
+              check-out.
+            </Text>
+            <Text>
+              Thank you again for choosing [Hotel Name]. We look forward to
```
Comment (iamhectorsosa) [inline src/emails/booking-confirmation.tsx:129]: This needs to be a var

**Comment by ericvalcik** [comment]
All the stuff that we discussed should be in this PR. 

- I've merged the `MinimalButton` to our Button component, but I believe there is still some room to improve the `Button` component. I found this website [buttons.cm](https://buttons.cm/) that even implements rounded corners for email buttons that work in outlook.
- The templates are all from the minimal theme, but I decided to drop the prefix, because its the only theme we currently have implemented. If we add more themes, we should figure out a way to separate the themes in the structure.

The templates were tested with [testi.at](testi.at) and everything shows up, but there are some discrepancies between the clients.

Let me know what you guys think @iamhectorsosa @jvorcak.

File: src/emails/news.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          <Row style={{ marginBottom: "32px" }}>
+            <Text style={{ fontSize: "48px", lineHeight: "52px" }}>
+              Stay Up-To-Date
+            </Text>
+          </Row>
+          <Row style={{ marginBottom: "16px" }}>
+            <Text>Dear Subscriber,</Text>
+            <Text>
+              We are excited to share the latest news and updates from [Your
+              Company/Brand Name]. As a valued member of our community, we want
```
Comment (iamhectorsosa) [inline src/emails/news.tsx:72]: var

**Review by iamhectorsosa — COMMENTED** [review]
I love these minimal templates! Great job! ❤️

File: src/emails/dinner-reservation.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+              experience. If you have any dietary requirements or special
+              requests, please let us know in advance.
+            </Text>
+            <Text>
+              Please note that your reservation will be held for 15 minutes past
+              the reserved time. If you anticipate arriving later than the
+              reserved time, please call {restaurantPhone} to inform us.
+            </Text>
+            <Text>
+              Thank you for choosing {restaurantName}. We look forward to
```
[the remark this answers is not in the document]
Comment (ericvalcik) [reply src/emails/dinner-reservation.tsx:106]: These two are already a var, or do you mean something else in the code?

[the remark this answers is not in the document]
Comment (iamhectorsosa) [reply src/emails/dinner-reservation.tsx:106]: You are right, my bad. I misread this one.

**Review by iamhectorsosa — APPROVED** [review]

**Pull request merged**